actions/generate-schedule/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/spinnaker/rotation-scheduler/actions/generate-schedule/scheduler"
+	"github.com/spinnaker/rotation-scheduler/actions/generate-schedule/users"
+	"github.com/spinnaker/rotation-scheduler/proto/schedule"
+)
+
+var (
+	// Required flags
+	stop              = flag.String("stop", "", "Stop generating the schedule after this date. Date is exclusive (not included in the schedule), and must be in the format 2006-01-02.")
+	shiftDurationDays = flag.Int("shiftDurationDays", 7, "The duration of each shift in days")
+
+	// One of these flags is required
```

Comment:
I might be missing something, but it feels like we should either:
1. not document that one of `start` and `previousSchedulePath` is required, since we fall back to a default instead of throwing an error when both are empty, or
2. document that one of `start` and `previousSchedulePath` is required, remove the default, and throw an error if both are empty in `validateFlags`?

Reply:
Remove the comment (so option 1, effectively).